{"name": "Test login attempt with wrong credentials", "status": "passed", "steps": [{"name": "test_login_incorrect", "status": "passed", "parameters": [{"name": "login_data", "value": "('alex_art', '')"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"1cfe4a509c9e9aaa829da199a4c1fbdd\")>"}], "start": 1537448449090, "stop": 1537448465532}], "parameters": [{"name": "login_data", "value": "('alex_art', '')"}], "start": 1537448449090, "stop": 1537448465532, "uuid": "c0025346-36ce-4111-aa3b-28a6a70f055c", "historyId": "bd8dff7e7869c90b8607f44062ef975e", "fullName": "src.tests.test_selenium.TestLogin#test_login_incorrect[login_data1]", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "Login"}, {"name": "tag", "value": "UI"}, {"name": "feature", "value": "User management"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "22160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}